index.html

```diff
@@ -815,7 +815,7 @@ <h3 id="message-format">Message format</h3>
             </tr>
             <tr>
                 <td>content</td>
-                <td>If the message is not encrypted, This is a dictionary containing free-form data for applications to interpret, plus a mandatory <em>type</em> field. The <em>type</em> field allows applications to filter out message types they don’t understand and must be a string between 3 and 52 characters long (inclusive). If this is a base64 encoded string, followed by a suffiix of <code>.box</code></td>
+                <td>If the message is not encrypted, This is a dictionary containing free-form data for applications to interpret, plus a mandatory <em>type</em> field. The <em>type</em> field allows applications to filter out message types they don’t understand and must be a UTF-16 string between 3 and 52 code units long (inclusive). If this is a base64 encoded string, followed by a suffiix of <code>.box</code></td>
```

Comment:
Can you fix the suffiiiiiix while you are at it? ;)

Reply:
oh, lol

Reply:
donez